RELEASE.md

```diff
@@ -62,44 +62,31 @@ $ git push --follow-tags
 
 ## Pre-releases
 
-### Automatic (recommended)
+to create betas 
 
-> [!TIP]
-> For detailed steps read [`beta-mode.sh`](./scripts/beta-mode.sh).
+from master checkout a new branch and run 
```

Comment:
unclear to me if that branch should have a specially formatted name (if yes I'd specify the format, if not I'd add that the name doesn't matter)